lib/factory.rb

```diff
@@ -0,0 +1,33 @@
+module Hotel_Factory
```

Comment:
Modules use capital `CamelCase`.

```suggestion
module HotelFactory
```